tests/lib/utils/utils-test.js

```diff
@@ -94,6 +94,14 @@ describe('isArrowFunctionExpression', () => {
   });
 });
 
+describe('isConciseArrowFunctionExpressionWithCall', () => {
+  const node = parse('test = () => foo()').right;
+
+  it('should check if node is concise arrow function expression with call expression body', () => {
+    assert.ok(utils.isConciseArrowFunctionWithCallExpression(node));
+  });
```

Comment:
I cannot help but mention that test case for `is _not* consise arrow function expression` would be nice to have - do you think it's redundant?